### PR TITLE
[notes] Update from previous comments.

### PR DESCRIPTION
**Pull request by arkhi**
## Summary of changes

I had this old comment from @tjoelsson bookmarked and finally took the time to update our style–guides accordingly.

### CODE REVIEW

File: language-javascript.md [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
+
+    ```js
+    var object = {
+        '$someVar': $someVar
+    };
+    ```
+
+- This is **not OK**:
+
+    ```js
```
Comment (tjoelsson) [inline language-javascript.md]: js is the same as javascript?

Comment (arkhi) [inline language-javascript.md]: Yes.

Comment (arkhi) [inline language-javascript.md]: It’s probably this rule: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1770

Comment (tjoelsson) [inline language-javascript.md]: Okay, just curious since both are used in this file. It's not a blocker.

**Comment by tjoelsson** [comment]
👍

**Comment by HouCoder** [comment]
🌾

**Comment by arkhi** [comment]
Thanks guys! Merging.

**Pull request merged**